.github/workflows/armv7l_wheels.yml

```diff
@@ -0,0 +1,65 @@
+name: armv7l wheels
+
+on:
+  push:
+  pull_request:
+  create:
+  schedule:
+    - cron:  '23 1 * * *'
+
+env:
+  KIVY_SPLIT_EXAMPLES: 1
+  SERVER_IP: '159.203.106.198'
+  DOCKER_IMAGE: 'balenalib/armv7hf-debian:buster'
+
+jobs:
+  armv7l:
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'schedule' || (github.event_name == 'create' && github.event.ref_type == 'tag') || contains(github.event.head_commit.message, '[build wheel]') || contains(github.event.head_commit.message, '[build wheel armv7l]') || contains(github.event.pull_request.title, '[build wheel]') || contains(github.event.pull_request.title, '[build wheel armv7l]')
+    steps:
+    - uses: actions/checkout@v1
+    - name: Make armv7l wheels
+      run: |
+        source .ci/ubuntu_ci.sh
+        generate_armv7l_wheels $DOCKER_IMAGE
+    - name: Rename wheels
+      if: github.event.ref_type != 'tag'
+      run: |
+        source .ci/ubuntu_ci.sh
+        rename_wheels
+    - name: Upload wheels to server
+      if: github.event_name != 'pull_request'
+      env:
+        UBUNTU_UPLOAD_KEY: ${{ secrets.UBUNTU_UPLOAD_KEY }}
+      run: |
+        source .ci/ubuntu_ci.sh
+        upload_file_to_server "$SERVER_IP" "armv7l/kivy/"
+    - name: Upload wheels as artifact
+      uses: actions/upload-artifact@master
+      if: github.event_name == 'pull_request'
+      with:
+        name: armv7l_wheels
+        path: dist
+    - name: Publish to PyPI
```

Comment:
I don't think we can upload armv71 wheels to pypi yet so please remove this section. Also, even if pypi does accept it, it's too early to do that yet. Maybe uploading to piwheels would be ok, if that is something we can do.

Reply:
Fixed in: 1678473d684941d4924c33cb7f8b478b9bf5de7c